pkg/asset/stock/stock.go

```diff
@@ -0,0 +1,27 @@
+package stock
+
+import (
+	"bufio"
+	"os"
+
+	"github.com/openshift/installer/pkg/asset/installconfig"
+)
+
+// Stock is the stock of assets that can be generated.
+type Stock struct {
+	installConfigStock
+}
+
+type installConfigStock struct {
```

Comment:
What's this struct for? Why not just put the installconfig.StockImpl inside the Stock directly? Seems a bit redundant to me if every package has to define this kind of struct too.

Reply:
OK, I see, it's embedding individual stocks into the master stock, so that we can call stock.InstallConfig() directly.

Reply:
Yes, it is so that `stock.Stock` can implement the `installconfig.Stock` interface without having to manually implement all of the functions while still allowing for implementing other Stock interfaces, such as `tls.Stock`.